src/core/bpf/socket_bind/socket-bind-api.bpf.h

```diff
@@ -0,0 +1,55 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+/* The SPDX header above is actually correct in claiming this was
+ * LGPL-2.1-or-later, because it is. Since the kernel doesn't consider that
+ * compatible with GPL we will claim this to be GPL however, which should be
+ * fine given that LGPL-2.1-or-later downgrades to GPL if needed.
+ */
+
+#include <linux/types.h>
+
+enum socket_bind_action {
+        SOCKET_BIND_DENY = 0,
+        SOCKET_BIND_ALLOW = 1,
+};
```

Comment:
The general recommendation is to keep BPF parts as close to kernel code style and C89 standard as possible. And so far everyone were pretty good about keeping that, so it would be great to keep systemd BPF code in the same consistent style.